packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -2295,6 +2297,8 @@
 	"workflows.empty.browseTemplates": "Explore workflow templates",
 	"workflows.empty.learnN8n": "Learn n8n",
 	"workflows.empty.button.disabled.tooltip": "Your current role in the project does not allow you to create workflows",
+	"workflows.empty.easyAI": "Test a ready-to-go AI Agent example",
+	"workflows.list.easyAI": "Test the power of AI in n8n with this ready-to-go News Agent Workflow",
```

Comment:
News?

Reply:
Good catch 🙌🏻

Reply:
Changing this to `AI` agent since the use-case may change